src/main/java/bio/terra/service/upgrade/UpgradeService.java

```diff
@@ -38,7 +38,7 @@ public UpgradeService(JobService jobService, ApplicationConfiguration appConfig,
     }
 
     public String upgrade(UpgradeModel request, AuthenticatedUserRequest user) {
-        // Make sure the user is a steward by checking for list jobs action
+        // Make sure the user is a steward or admin by checking for list jobs action
```

Comment:
It seems like we should have an IAM action for database admin operations. This is the exactly kind of thing you're not supposed to do with permissions, e.g., use one permission as a proxy for another.

Reply:
Hm, yeah seems like a good idea. We're not actually using this chunk of code - there's just a skeleton to implement a one-off flight.